backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -29,7 +25,7 @@ public class CrdtMiniLcmApi(
     CurrentProjectService projectService,
     IMiniLcmCultureProvider cultureProvider,
```

Comment:
This parameter appears to be unused now that cultureProvider code has moved over to MiniLcmRepository.